plutus-metatheory/src/Untyped.lagda.md

```diff
@@ -40,6 +40,12 @@ open Untyped
 
 ## Well-scoped Syntax
 
+This defines the syntax for UPLC and requires that it be "well scoped", which
+is that only variables in the context are used. The context uses de Bruijn naming,
+so the variables are numbered. This numbering is provided by an indctivley defined
+natural number, which uses the Maybe type (so: `Nothing` = zero, `Just Just Nothing` = 2)
```

Comment:
```suggestion
natural number, which uses the Maybe type (so: `Nothing` = zero, `Just (Just Nothing)` = 2)
```

Perhaps it's also worth referencing [de Bruijn notation as a nested datatype](https://www.cambridge.org/core/services/aop-cambridge-core/content/view/D8BFA383FDA7EA3DC443B4C42A168F30/S0956796899003366a.pdf/de-bruijn-notation-as-a-nested-datatype.pdf) for a detailed explanation of the trick.